RiotNSE/NSEMemoryStore.swift

```diff
@@ -89,9 +91,20 @@ class NSEMemoryStore: MXMemoryStore {
         return MXUser(userId: userId)
     }
     
+    override var syncFilterId: String? {
+        get {
+            let filter = MXFilterJSONModel()
+            filter.room = MXRoomFilter()
+            filter.room.rooms = []
+            return filter.jsonString()
+        } set {
+            //  no-op
+        }
+    }
+    
     override func close() {
         //  close real store
-        fileStore.close()
+        NSEMemoryStore.fileStore.close()
```

Comment:
This means the store will be closed after the handling of the first push. What about other pushes in the queue?
It seems hard to fix at this level. The only option I see is a static `MXSession`.

Reply:
It seems `-[MXFileStore close]` does nothing serious, as no pending commits. I'll totally remove this call.

Reply:
Now i'm more concerned about `-[MXSession close]` calls.